src/main/java/ConsoleOutputView.java

```diff
@@ -0,0 +1,25 @@
+import java.math.BigDecimal;
+import java.math.RoundingMode;
+import java.util.Arrays;
+import java.util.Collections;
+
+public class ConsoleOutputView {
+    public void view(Lotto lotto) {
+        System.out.println(lotto.size() + "개를 구매했습니다.");
+        for (LottoNumbers lottoNumbers : lotto) {
+            System.out.println(lottoNumbers);
+        }
+        System.out.println();
+    }
+
+    public void view(Aggregator aggregator) {
+        System.out.println();
+        System.out.println("당첨 통계");
+        System.out.println("---------");
+        Arrays.stream(Prize.values()).sorted(Collections.reverseOrder()).filter(Prize::win).forEach(prize -> {
+            System.out.println(prize + " - " + aggregator.countGroupBy(prize) + "개");
+        });
+        BigDecimal bd = aggregator.yield().setScale(2, RoundingMode.DOWN);
```

Comment:
`yield()` 에서 `setScale`을 해서 리턴해주면 어떨까요?

데이터를 가져와서 추가적으로 제어 하는것보다 책임을 좀 더 부여하면 깔끔해질것 같다는 생각이 드네요  :)

아실수도 있지만 도움될만한 블로그 링크입니다 ^^([디미터법칙](https://tecoble.techcourse.co.kr/post/2020-06-02-law-of-demeter/))

Reply:
@wooobo 
```suggestion
        BigDecimal yield = aggregator.yield();
        yield.setScale(2, RoundingMode.DOWN);
        System.out.println("총 수익률은 " + yield.doubleValue() + "입니다.");
```

혹시 이렇게 표현하게 되면 괜찮을까요? 
BigDecimal 을 java 에서 정밀하게 숫자를 다룰 때 사용할 수 있는 객체 자료형이라고 생각했습니다. 
`aggregator 가 수익률 데이터를 BigDecimal 로 뷰에게 전달`하면,
 `뷰에서는 뷰 정책에 맞게 소숫점 2자리까지 내림 해서 보여준다`. 
요렇게 뷰와 aggregator 의 책임을 각각 가져가고자 합니다. 👀